packages/app-shell/src/common/hooks/useModal.ts

```
@@ -79,9 +79,13 @@ function useModal({ refetchAccountData }: UseModalProps): UseModalReturn {
   }
 
   useEffect(() => {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```

I'm ignoring the errors on the event listeners for now - we know they work, and we will likely get rid of these anyway if we directly consume the app-shell in buffer-mono